integration_tests/integration/cancelVisit.cy.ts

```diff
@@ -38,7 +38,7 @@ context('Cancel visit journey', () => {
 
     cy.task('stubPrisonerById', prisoner)
     cy.task('stubVisitHistory', visitHistoryDetails)
-    cy.task('stubPrisonerSocialContacts', { offenderNo, contacts })
+    cy.task('stubPrisonerSocialContacts', { offenderNo, contacts, approvedVisitorsOnly: false })
```

Comment:
Unrelated to this PR, but here and elsewhere fixed various calls to `stubPrisonerSocialContacts` that have been failing silently (i.e. logging errors but tests still passing) since #799 when a new parameter was introduced.